static/js/list_render.js

```diff
@@ -264,7 +264,11 @@ exports.create = function ($container, list, opts) {
     };
 
     widget.set_up_event_handlers = function () {
-        meta.scroll_container = scroll_util.get_list_scrolling_container($container);
+        if (opts.simplebar_container) {
+            meta.scroll_container = ui.get_scroll_element(opts.simplebar_container);
+        } else {
+            meta.scroll_container = scroll_util.get_list_scrolling_container($container);
+        }
```

Comment:
@andersk what are your thoughts on this commit?  I can see how explicit might be better than implicit, but I'm uncertain that this is the right strategy for being explicit.

Reply:
`list_render` already has too many variants, do we have to add more? I would think that we either want SimpleBar on all of our `list_render` instances or none of them.

Reply:
Yeah, I sorta feel the same way.  I'm not aware of `list_render` instances where we don't want simplebar, though I suppose it could be annoying to audit them all.

In any case @amanagr can you pull this commit out to a separate PR?

Reply:
I pushed to commit in https://github.com/zulip/zulip/pull/15673 @sahil839 FYI

Reply:
I left the commit here in case you want to test-deploy.